.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 extension
-extension.zip
+.DS_Store
```

Comment:
.DS_Store should not be ignored in project gitignores, only your global gitignore (see [Configuring ignored files for all repositories on your computer](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer))

Reply:
Why is that?

Reply:
Because these files are system. There are many different files that could be created by systems, editors, etc. It's better for projects to only ignore files that all users will have regardless of their system (such as node_modules in Node projects) as we won't be able to predict what machines/editors/etc. are being used by contributors.

Reply:
Fair enough! I don't think it does much harm, but totally get your point :+1: